src/Type/Php/ArrayFilterFunctionReturnTypeReturnTypeExtension.php

```diff
@@ -71,6 +74,9 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 				} elseif ($callbackArg instanceof ArrowFunction && count($callbackArg->params) > 0) {
 					$var = $callbackArg->params[0]->var;
 					$expr = $callbackArg->expr;
+				} elseif ($callbackArg instanceof String_) {
```

Comment:
This looks how I would have imagined it - good job

Reply:
Thx, yeah, I'll add more tests for sure.